src/StoreProvider.lua

```diff
@@ -4,18 +4,21 @@ local storeKey = require(script.Parent.storeKey)
 
 local StoreProvider = Roact.Component:extend("StoreProvider")
 
-function StoreProvider:init(props)
+function StoreProvider.validateProps(props)
 	local store = props.store
-
 	if store == nil then
-		error("Error initializing StoreProvider. Expected a `store` prop to be a Rodux store.")
+		return false, "Error initializing StoreProvider. Expected a `store` prop to be a Rodux store."
 	end
+	return true
+end
 
+function StoreProvider:init(props)
+	local store = props.store
 	self._context[storeKey] = store
 end
 
 function StoreProvider:render()
-	return Roact.oneChild(self.props[Roact.Children])
+	return Roact.createFragment(self.props[Roact.Children])
```

Comment:
It looks like one of your other changes got tangled up here. Can you make sure your changes are limited to just this ticket in this PR?